packages/vision/package.json

```diff
@@ -1,42 +1,16 @@
 {
+  "repository": "GoogleCloudPlatform/google-cloud-node",
   "name": "@google-cloud/vision",
-  "version": "0.11.3",
-  "author": "Google Inc.",
-  "description": "Cloud Vision Client Library for Node.js",
-  "contributors": [
-    {
-      "name": "Burcu Dogan",
-      "email": "jbd@google.com"
-    },
-    {
-      "name": "Johan Euphrosine",
-      "email": "proppy@google.com"
-    },
-    {
-      "name": "Patrick Costello",
-      "email": "pcostell@google.com"
-    },
-    {
-      "name": "Ryan Seys",
-      "email": "ryan@ryanseys.com"
-    },
-    {
-      "name": "Silvano Luciani",
-      "email": "silvano@google.com"
-    },
-    {
-      "name": "Stephen Sawchuk",
-      "email": "sawchuk@gmail.com"
-    }
-  ],
-  "main": "./src/index.js",
+  "version": "0.7.1",
```

Comment:
The version should remain the same as it is, 0.11.3. When we publish after the merge, it will become 0.12.0.